src/utils/thumbnailsUtils.js

```diff
@@ -0,0 +1,22 @@
+import { fetchApiCall } from './api.js'
+import { useConfigurationStore } from '../stores/configuration.js'
+
+const getThumbnails = async (params, queryValue) => {
```

Comment:
This can only take a single param and the queryValue for that param, so maybe call it `param` instead of `params` to avoid confusion?

Reply:
Alternatively, you could have it just take `params` which is a string with any query params you want to pass in to be more flexible, so it could be `param1=value1&param2=value2` if you wanted to be able to pass multiple params.